### ISSUE TITLE
Cannot Create/Update `user_fields` attribute on users

### ISSUE THREAD

**Issue by KidA001** [lanej/zendesk2#37]
When calling: 
```
client.users.create!(
  name: 'Some Name',
  email: 'user@user.net',
  user_fields: { 'user_type' => 'pro' }
)
```

`user_fields` never gets set in zendesk, even though we have a custom field for the user.

**Comment by lanej** [comment]
@KidA001 if i remember correctly the `user_fields` hash need to have the field id as the key.

**Comment by lanej** [comment]
jk that's ticket fields

**Comment by lanej** [comment]
Fixed in v1.7.2
